tests/utils/test_status.py

```diff
@@ -158,10 +158,10 @@ def test_str(mock_api_issue):
         report = PipelineStatusReport()
         report.generate()
         print(report)
-        test_string = '    "2T6S": {\n        "softwares": "SPM",\n        "fmriprep": "Yes",'
+        test_string = '    "2T6S": {\n        "software": "SPM",\n        "fmriprep": "Yes",'
         test_string += '\n        "issues": {},\n        "status": "done"\n    },'
         assert test_string in str(report)
-        test_string = '    "C88N": {\n        "softwares": "SPM",\n        "fmriprep": "Yes",'
+        test_string = '    "C88N": {\n        "software": "SPM",\n        "fmriprep": "Yes",'
```

Comment:
Do not change softwares here, as this is a dictionary key

Reply:
probably easier to add `softwares` to the list of ignored words in the codespell config